be/src/vec/functions/array/function_array_except.cpp

```diff
@@ -0,0 +1,79 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "vec/functions/array/function_array_binary.h"
+#include "vec/functions/array/function_array_set.h"
+
+#include "vec/functions/simple_function_factory.h"
+
+namespace doris::vectorized {
+
+struct NameArrayExcept {
+    static constexpr auto name = "array_except";
+};
+
+template <typename Set, typename Element>
+struct ExceptAction {
+    // True if set has null element
+    bool null_flag = false;
+    // True if result_set has null element
+    bool result_null_flag = false;
+    // True if it should apply the left array first.
+    static constexpr auto apply_left_first = false;
+
+    // Handle Null element.
+    // Return ture means this null element should put into result column.
+    bool apply_null(bool left_or_right) {
+        if (left_or_right) {
```

Comment:
Is `left_or_right` always equal to `apply_left_first` ? If it is true, we do not need to parse `left_or_right` param.

Reply:
No. 
If `apply_left_first` = true, 
means apply_left first, then `left_or_right` = true
and apply_right second, then `left_or_right` = false.
So actually the value of `left_or_right` is true at first, then is become false.